org.eclipse.winery.frontends/app/tosca-management/src/app/wineryLicenseModule/wineryLicense.component.html

```diff
@@ -11,50 +11,174 @@
+                                   (keyup)="search($event.target)" autocomplete="off" placeholder="search">
+                            <mat-divider></mat-divider>
+                            <mat-option *ngFor="let item of selectedOptions" [value]="item"
+                                        (click)="dropdownAction(item);showForm=false">
+                                {{item}}
+                            </mat-option>
+                        </mat-select>
+                    </mat-form-field>
+                </form>
+                <div [class.hidden]="!loadingLicense" style="align-self: center">
+                    <winery-loader loaderSize="30"></winery-loader>
+                </div>
+            </div>
+
+            <div class="btn-right">
+                <button class="btn btn-sm btn-primary middle" (click)="reset()" *ngIf="licenseEngine">
+                    Check License
+                </button>
+                <button class="btn fa fa-floppy-o" aria-hidden="true" name="save"
+                        [disabled]="!currentLicenseText || showForm"
+                        (click)="saveConfirm()"> Save
+                </button>
+                <button class="btn fa fa-ban" name="cancel" aria-hidden="true" (click)="cancelEdit()"> Cancel</button>
+            </div>
+        </div>
+
+        <textarea [(ngModel)]="currentLicenseText" *ngIf="!showForm"></textarea>
+
+        <div *ngIf="showForm" style="clear: both">
+            <mat-horizontal-stepper [linear]="true" #stepper>
+                <mat-step completed="false">
+                    <form [formGroup]="firstFormGroup">
+                        <ng-template matStepLabel>Fetch and Analyze Licenses</ng-template>
+                        <div class="inputFields">
+                            <mat-form-field style="width: 65%; font-size: 15px">
+                                <input matInput formControlName="link" required>
+                                <mat-placeholder class="placeholder">Repository URL</mat-placeholder>
+                            </mat-form-field>
+                            <mat-form-field style="width: 35%; font-size: 15px">
+                                <input matInput formControlName="branch">
+                                <mat-placeholder class="placeholder">Branch <span style="font-size: 12px">(if omitted, default branch is used)</span>
+                                </mat-placeholder>
+                            </mat-form-field>
+                        </div>
+                        <div *ngIf="loadingbar">
+                            <mat-progress-bar mode="indeterminate"
+                                              *ngIf="!isFailed()"></mat-progress-bar>
+                            <mat-progress-bar mode="determinate" value="50" color="warn"
+                                              *ngIf="isFailed()"></mat-progress-bar>
+                            <div style="float: right">Status: {{getStatus()}}</div>
```

Comment:
Does this update automatically? In my experience, Angular will not detect any change from methods but only variables.

Reply:
It works with a method